cmd/run.go

```diff
@@ -90,6 +90,13 @@ Non-bundle data file and directory paths can be prefixed with the desired
 destination in the data document with the following syntax:
 
 	<dotted-path>:<file-path>
+		
+To set a data file as the input document in the interactive shell prefix the file path:
+	
+	opa run repl.input:input.json
```

Comment:
Ok, one more adjustment after re-reading again (Sorry for the nit-picking 😢 this is very prominently user facing so we want it to be well polished)

Lets change to:

```
To set a data file as the input document in the interactive shell use the
"repl.input" path prefix with the input file:
	
	repl.input:<file-path>

Example:

	opa run repl.input:input.json

Which will load the "input.json" file at path "data.repl.input".

Use the "help input" command in the interactive shell to see more options.
```

Which changes a couple of things:
* We match up with the formatting from the previous prefix explanation, but keep an example.
* Line breaks at 80 characters for some standardization with the console help output

Reply:
ok @patrick-east, no problem, I want it to be the best too 👍

Reply:
You can look at e46538 for the fix.